### PR TITLE
Add some checks in SimpleGBLTrajAliDriver  etc

### PR DESCRIPTION
**Pull request by bloodyyugo**
In SimpleGBLTrajAliDriver:  
Add a check that trackGblStripClusterData is filled. 
Checks that global derivatives don't contain a NaN. 

In GBLOutputDriver:
Change range on Chi2oNDF so that details can be seen

### CODE REVIEW

File: tracking/src/main/java/org/hps/recon/tracking/gbl/SimpleGBLTrajAliDriver.java [hunk 1]
```diff
@@ -772,10 +772,13 @@ else if (trackSide == 1 && !TrackUtils.isHoleTrack(track))
             
             List<GBLStripClusterData> trackGblStripClusterData  = computeGBLStripClusterData(track,TrackType,
                                                                                              temp,gblStripClusterDataRelations,event);
-            
-            //Printout the Cluster Data: 
-            
-            
+
+	    if(trackGblStripClusterData == null){
+		System.out.println("SimpleGBLTrajAliDriver::trackGblStripClusterData is null, skipping track");
```
Comment (pbutti) [inline tracking/src/main/java/org/hps/recon/tracking/gbl/SimpleGBLTrajAliDriver.java:783]: @bloodyyugo
Remove println, or put it under debug pls.

File: tracking/src/main/java/org/hps/recon/tracking/gbl/SimpleGBLTrajAliDriver.java [hunk 2]
```diff
@@ -809,17 +812,25 @@ else if (trackSide == 1 && !TrackUtils.isHoleTrack(track))
...
-                        ComputeStructureDerivatives(gblpoint);
-                    else
-                        ComputeCOMDerivatives(gblpoint);
-                    
-                }//point loop
-                
+                    if (!doCOMAlignment){
+                        isGood=ComputeStructureDerivatives(gblpoint);
+			if(!isGood){
+			    System.out.println("SimpleGBLTrajAli::ComputeStructureDerivatives globals derivatives probably have NaN");
```
Comment (pbutti) [inline tracking/src/main/java/org/hps/recon/tracking/gbl/SimpleGBLTrajAliDriver.java:826]: Remove println or put it under debug

File: tracking/src/main/java/org/hps/recon/tracking/gbl/SimpleGBLTrajAliDriver.java [hunk 3]
```diff
@@ -809,17 +812,25 @@ else if (trackSide == 1 && !TrackUtils.isHoleTrack(track))
...
+			    break; 
+			}
+		    }
+                    else{
+                        isGood=ComputeCOMDerivatives(gblpoint);
+			if(!isGood){
+			    System.out.println("SimpleGBLTrajAli::ComputeCOMDerivatives globals derivatives probably have NaN");
```
Comment (pbutti) [inline tracking/src/main/java/org/hps/recon/tracking/gbl/SimpleGBLTrajAliDriver.java:833]: Remove println or put it under debug

**Review by pbutti — CHANGES_REQUESTED** [review]

**Comment by cbravo135** [comment]
detector.properties has not been updated properly for most of these detectors. Not sure we want to add every iteration you did while aligning to the master branch. We should chat about what all from this PR we want to include.

**Pull request closed without merging**